global.json

```diff
@@ -4,6 +4,11 @@
   },
   "tools": {
     "dotnet": "6.0.200",
+    "runtimes": {
+        "dotnet": [
+          "3.1.0"
```

Comment:
This is fine, but please consider configuring the tool for rollforward/major so it doesn't impose this requirement on other repos.

Reply:
I created an issue for the manifest tool folks to add it https://github.com/microsoft/dropvalidator/issues/454, but I did not hear back from them, so I created a workaround

Reply:
This change has broken the build on macOS with Apple Silicon (arm64).

```
dotnet-install: The resource at legacy link 'https://dotnetbuilds.azureedge.net/public/Runtime/3.1.0/dotnet-osx-arm64.3.1.0.tar.gz' is not available.
dotnet_install: Error: Could not find `.NET Core Runtime` with version = 3.1.0
dotnet_install: Error: Refer to: https://aka.ms/dotnet-os-lifecycle for information on .NET Core support
```

Reply:
Opened https://github.com/dotnet/msbuild/issues/7834

Reply:
@BenVillalobos, is there some way to work around #7834 without either reverting this or making the arcade change rainersigwald suggested?

The arcade change is my favorite option. If there's no workaround for jrdodds, I'd favor reverting this; having to ignore the sbom thing every time is annoying, but at least we can get around it.

Reply:
> is there some way to work around https://github.com/dotnet/msbuild/issues/7834 without either reverting this or making the arcade change rainersigwald suggested?

Not that I'm aware of. Everything under "tools" seems to be an arcade concept. 

> If there's no workaround for jrdodds, I'd favor reverting this

Fully agree. We're officially supporting arm64 now, we should treat "working arm64 builds" as a requirement. It's better that we bite the bullet and continue doing what we've been doing with the sbom stuff.

> I was wondering if there's something we can do that would let our official builds pass and create insertions PRs with SBoM without requiring a change on an arm64 machine before building.

That'd require a split in our main branches, unfortunately that's a no-go without some overly complicated git magic :(